### ISSUE TITLE
CS0114 and CS0109 on Mac and iOS, Control/UIView has properties other platforms do not.   

### ISSUE THREAD

**Issue by Arlodotexe** [unoplatform/uno#15252]
### Current behavior

Following up on https://github.com/CommunityToolkit/Windows/issues/248, when we were porting the WCT for 8.0, we found several warnings about existing properties when we inherit from `Control`, but only under iOS and Mac TFMs. 

The affected properties in the toolkit have been suppressed for now (see [here](https://github.com/search?q=repo%3ACommunityToolkit%2FWindows+%28CS0109+OR+CS0114%29&type=code) for reference). 

### Expected behavior

Behavior when inheriting from `Control` should be consistent across platforms, ideally not requiring warning suppression.

### How to reproduce it (as minimally and precisely as possible)

1. Set up a new Uno project, make sure TargetFrameworks `net7.0-ios` and `net7.0-maccatalyst` are included, plus at least one other TFM for comparison.
2. Create a new class that inherits from `Control`.
3. Create a property in the new class named ScaleX, Transform, or Description. 
4. Build. Observe CS0114 or CS0109 under the ios and mac TFM, but not the others.

### Works on UWP/WinUI

Yes

### Affected platforms

iOS, Mac Catalyst

### IDE

Visual Studio 2022

**Comment by Youssef1313** [comment]
This is looking by-design. These properties come from iOS' UIView which we can't control. I don't see anything we can do other than providing a `DiagnosticSuppressor`, but that doesn't look like a good solution as well. The warning is true, so a diagnostic suppressor may not be the right thing to do.

@jeromelaban thoughts?

**Comment by jeromelaban** [comment]
Indeed, this is not something we control for iOS/Android/Catalyst. This is not happening for Wasm and Skia targets, though.
